### ISSUE TITLE
[Enhancement] `se titlecase` usage note in Step-by-Step guide

### ISSUE THREAD

**Issue by dajare** [standardebooks/web#206]
**tl;dr**

- please add information on the `-n` switch for `se titlecase` to the Step-by-Step Guide;
- relevant location is @ [line 183 of the page code][1].

---

The last project I attempted gave me, for the first time, chapter titles in ALL CAPS.

I have an editor`¹` that makes it very simple to filter selected text through an external command, so used `se titlecase` for this. It was so annoying to have selected the text between the `<h3>...</h3>` tags, only to have the closing tag shunted on to the next line for every chapter title.

I thought it was a glitch, so looked at the code on Github and ... discovered there is a `--no-newline -n` switch! Wonderful! (I note, too, this is nicely explained in the `--help` text.)

Given that the [use case as presented in the S-b-S guide is for chapter titles][1], would it be worth alerting producers to this at that point? That would have helped me. At least there were only 28 chapters ... not 80. :wink:

`¹` That was [Textadept][t], but I've since worked out how to manage this in [VS Codium][v] (assigned to shortcut key) which I normally use for SE projects.

[1]: https://github.com/standardebooks/web/blob/75fd962088c6390f3307d6f7d2aae85bd168dbdd/www/contribute/producing-an-ebook-step-by-step.php#LL183-L183C200
[t]: https://orbitalquark.github.io/textadept/
[v]: https://vscodium.com/

**Comment by acabal** [comment]
Sure, you create a pull request to mention that if you like.

**Comment by dajare** [comment]
Pull request created - I'm confident you can improve my wording! Hope this helps. / David
